array.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * This function checks if an array is a factor chain or not
+ * Example of factor array - [2,4,8,16]
```

Comment:
It would be wonderful to add a proper formatting to array elements, do the same to the line below

Reply:
Sure, formatted the code and added additional comments